docs/developers/parallel_transfer.md

```diff
@@ -0,0 +1,44 @@
+#
+
+## Parallel Transfer
+
+As of iRODS 4.3.4, parallel transfer via the high ports is deprecated. Future versions of iRODS will use the zone port for parallel transfer. Therefore, this page is dedicated to everything related to parallel transfer via the zone port.
+
+### Why deprecate parallel transfer over the high ports?
+
+There are a few reasons for why this functionality is deprecated:
+
+- High ports required network administrators to open a wide range of ports
+- Implementation required special code paths in the server, making it complex to maintain and reason about
+- Only provided via the Put API endpoint
```

Comment:
Get API also uses it. And everything else in server-to-server transfers (Copy, Repl, etc.), but those aren't as important.

Reply:
Done.